charts/datarepo-ui/templates/configmap.yaml

```diff
@@ -25,6 +25,16 @@ data:
       server { # simple reverse-proxy
         listen       {{ .Values.nginxport }};
 
+        # add a Content-Security-Policy, with some variables to make formatting nice
+        set $DEFAULT "default-src 'self'";
+        set $SCRIPT "script-src 'self' 'unsafe-inline' apis.google.com *.gstatic.com *.broadinstitute.org";
+        set $STYLE "style-src 'self' 'unsafe-inline' fonts.googleapis.com";
+        set $IMG "img-src 'self' lh3.googleusercontent.com";
+        set $FONT "font-src 'self' fonts.googleapis.com fonts.gstatic.com";
+        set $CONNECT "connect-src 'self' *.broadinstitute.org account.google.com";
```

Comment:
Should we also allow scripts and connections from `*.terra.bio` , too? I think it's needed for our testing and prod environments.

Reply:
I'll check out prod and add things as necessary. Thanks for the catch!

Reply:
Checked prod, and I don't see it hitting anything in `*.terra.bio`, only `*.broadinstitute.org`